addons/web/static/tests/core/position_hook_tests.js

```diff
@@ -176,6 +177,24 @@ QUnit.test("position right", getPositionTest("right"));
 QUnit.test("position top-start", getPositionTest("top-start"));
 QUnit.test("position top-middle", getPositionTest("top-middle"));
 QUnit.test("position top-end", getPositionTest("top-end"));
+QUnit.test("position RTL top-start", async function (assert) {
+    patchWithCleanup(localization, {
+        direction: "rtl",
+    });
+    await getPositionTest("top-start", "top-end")(assert);
+});
+QUnit.test("position RTL top-middle", async function (assert) {
+    patchWithCleanup(localization, {
+        direction: "rtl",
+    });
+    await getPositionTest("top-middle")(assert);
+});
+QUnit.test("position RTL top-end", async function (assert) {
+    patchWithCleanup(localization, {
+        direction: "rtl",
+    });
+    await getPositionTest("top-end", "top-start")(assert);
+});
```

Comment:
As this test suite is already exhaustive, I would like to see it evolve in the same way.
Add tests for:
- bottom-{start,middle,end}
- right-{start,middle,end}
- left-{start,middle,end}